shared-libs/cht-datasource/src/local/person.ts

```diff
@@ -58,4 +58,22 @@ export namespace v1 {
       return deepCopy(personWithLineage);
     };
   };
+
+  /** @internal */
+  export const getPage = ({ medicDb, settings }: LocalDataContext) => {
+    const personIdentifierRecord = contactTypeUtils.getPersonTypes(settings.getAll());
+
+    let personIdentifier: string;
+    if (Object.entries(personIdentifierRecord).length > 0) {
+      personIdentifier = personIdentifierRecord[0]?.id as string;
+    } else {
+      throw new Error('Person type not found');
+    }
+
+    const getDocsByPage = queryDocsByKey(medicDb, 'medic-client/contacts_by_type');
+
+    return async (limit: number, skip: number): Promise<Nullable<Doc>[]> => {
```

Comment:
The array returned by `queryDocsByKey` might contain null values and this is fine since sometimes those may have semantic meaning in the context of the query.  However, for this case, we can eliminate any null values (and anything that is not actually a person) by filtering the returned values via `isPerson`.  The return value for this function should actually just be `Promise<Person.v1.Person[]>`.